service/history/queue/interface.go

```diff
@@ -38,50 +38,56 @@ type (
 		ReverseMatch bool
 	}
 
-	// JobInfo indicates the scope of a Job and its progress
-	JobInfo interface {
-		QueueID() int
+	// ProcessingQueueStates is a list of ProcessingQueueState
+	ProcessingQueueStates []ProcessingQueueState
+
+	// ProcessingQueueState indicates the scope of a task processing queue and its current progress
+	ProcessingQueueState interface {
+		CollectionID() int
 		MinLevel() task.Key
 		MaxLevel() task.Key
 		AckLevel() task.Key
 		ReadLevel() task.Key
 		DomainFilter() DomainFilter
 	}
 
-	// Job is responsible for keeping track of the state of tasks
+	// ProcessingQueues is a list of ProcessingQueue
+	ProcessingQueues []ProcessingQueue
+
+	// ProcessingQueue is responsible for keeping track of the state of tasks
```

Comment:
This comment needs to be updated with new names

Reply:
Done.